Description: Fixes the compatibility with the version of Plexus Containers used in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/JdkToolchainConverter.java
+++ b/animal-sniffer-maven-plugin/src/main/java/org/codehaus/mojo/animal_sniffer/maven/JdkToolchainConverter.java
@@ -80,7 +80,7 @@
                 String val = params[j].getValue();
                 parameters.put( name, val );
             }
-            catch ( PlexusConfigurationException ex )
+            catch ( Exception ex )
             {
                 throw new ComponentConfigurationException( ex );
             }
